tasks/Module 2/task_1.py

```diff
@@ -0,0 +1,46 @@
+def display_inventory(inventory):
+    print("Inventory:")
+    item_total = 0
+    for item_name, item_count in inventory.items():
+        print(item_count, item_name)
+        item_total += item_count
+    print("Total number of items: " + str(item_total))
+    if item_total >= 80:
+        print("CAUTION: You are overloaded, can't move!")
+    elif item_total >= 70:
+        print("CAUTION: Your equipment is very heavy, you're moving slower than usual!")
+    elif item_total >= 60:
+        print("CAUTION: Your backpack weighs a lot, your stamina runs out quicker!")
+
+
+def add_to_inventory(inventory, added_items):
+    skipped = {}
+    added_count = 0
+    for item_name in added_items:
+        if item_name in ['chewed gum', 'rubbish', 'used tissue']:
+            if item_name in skipped.keys():
+                skipped[item_name] += 1
+            else:
+                skipped[item_name] = 1;
+        else:
+            if item_name in inventory.keys():
+                inventory[item_name] += 1
+            else:
+                inventory[item_name] = 1
+            added_count += 1
+    print("Added " + str(added_count) + " items to the inventory")
```

Comment:
As we are using Python 3.7 or newer, we can make good usage out of fstrings.
They improve readability and simplify things a lot.

https://realpython.com/python-f-strings/